k-distribution/k-tutorial/1_basic/04_disambiguation/README.md

```diff
@@ -316,12 +315,24 @@ modify the definition so that you will get the alternative parse.
 2. Modify your solution from Lesson 1.3, Exercise 2 so that unary negation should
 bind tighter than multiplication and division, which should bind tighter than
 addition and subtraction, and each binary operator should be left associative.
-Write these priority and associativity declarations both inline and explicitly.
+Write these priority and associativity declarations explicitly and then
+try to think whether they can be written inline.
```

Comment:
To me, it sounds like you want to make the inline part optional.

Reply:
I'd like to keep it mandatory without telling the reader it's actually possible.
How about the following?

```
Write these priority and associativity declarations explicitly and then
answer whether they can be written inline.
``` 

Not that important a change though, we can drop it :-)

Reply:
How about
```
Write these priority and associativity declarations explicitly, and then
try to write them inline.
```

Reply:
Works for me!